engine.py

```diff
@@ -223,30 +235,20 @@ def init_engine_nuke(self):
         # Store data needed for bootstrapping Tank in env vars. Used in startup/menu.py.
         os.environ["TANK_NUKE_ENGINE_INIT_NAME"] = self.instance_name
         os.environ["TANK_NUKE_ENGINE_INIT_CONTEXT"] = tank.context.serialize(self.context)
-        os.environ["TANK_NUKE_ENGINE_INIT_PROJECT_ROOT"] = self.tank.project_path
-        
-        # Add our startup path to the nuke init path
-        startup_path = os.path.abspath(os.path.join(os.path.dirname(__file__), "startup"))
-        tank.util.append_path_to_env_var("NUKE_PATH", startup_path)        
-    
-        # We also need to pass the path to the python folder down to the init script
-        # because nuke python does not have a __file__ attribute for that file.
-        local_python_path = os.path.abspath(os.path.join(os.path.dirname(__file__), "python"))
-        os.environ["TANK_NUKE_ENGINE_MOD_PATH"] = local_python_path
 
-    def pre_app_init(self):
-        """
-        Called at startup, but after QT has been initialized.
-        """
-        if self.hiero_enabled or self.studio_enabled:
-            return
+        # If we're in Toolkit classic mode, we need to backup a few things in order to be able
+        # to restart the engine after a File->New|Open
+        if not self.in_plugin_mode:
+            os.environ["TANK_NUKE_ENGINE_INIT_PROJECT_ROOT"] = self.tank.project_path
```

Comment:
They are not. They look new because the diff is bad. These are for classic mode.